### DISCUSSION TITLE
r/city-china-zhejiang-shaoxing-xinchang

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#41] [Announcements]
# r/city-china-zhejiang-shaoxing-xinchang

游记｜新昌，不只是碳水王国

https://blog.ops-coffee.cn/r/city-china-zhejiang-shaoxing-xinchang

**Comment by marvyn** [comment]
新昌的确不错，元旦也从上海去了这里，躲过了上海的严重雾霾，体验特别好。
第一天到达镜岭，第二天爬十九峰，风景很好。

晚上还庆祝跨年放了烟花。

第三天也去了岩贝村，的确灰蒙蒙的。

这次没去县城和大佛那边，你提到的这两个地方看上去也挺不错，下次再去~~

**Reply by ops-coffee** [reply]
😄竟然没有偶遇在下岩贝

**Comment by linchanx** [comment]
看起来相当好吃的样子

**Reply by ops-coffee** [reply]
我觉得还阔以，去试试
